### PR TITLE
Feat: Added Product Listing with Filters

### PR DESCRIPTION
**Pull request by shuntzhere**
Added product listing page 
Adde the filters.

Screenshot (37)

### CODE REVIEW

File: css/listings.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+.listings__sidebar--title {
+  font-family: "Cinzel", serif;
+  font-size: 2rem;
+  font-weight: var(--font-regular);
+  margin: -1rem 0 0 0;
+}
+
+.form__group {
+  padding: 0.2rem;
```
Comment (Priyanshi-Tiwari) [inline css/listings.css:37]: You can use pixels for values less than 1rem. Everything else looks good.

**Review by Priyanshi-Tiwari — APPROVED** [review]

File: pages/listings.html [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/55aa2069c0.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="/main.css" />
+    <title>Product List</title>
+  </head>
+  <body>
+    <div class="container">
```
Comment (ttahm3d) [inline pages/listings.html:15]: Remove this `div` if it is not doing anything.

File: pages/listings.html [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+              <span>t</span>
+              <span>i</span>
+              <span>c</span>
+              <span>k</span>
+              <span>o.</span>
+            </a>
+          </div>
+
+          <div class="header__nav--search">
+            <i class="fas fa-search fa-2x" style="color: black"></i>
```
Comment (ttahm3d) [inline pages/listings.html:30]: Remove inline style

File: css/listings.css [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+  padding: 1rem;
+}
+
+.listing__main {
+  width: 100%;
+  display: grid;
+  padding: 1rem;
+  grid-template-columns: repeat(auto-fill, minmax(18rem, 1fr));
+  gap: 1.5rem;
+  overflow: auto;
```
Comment (ttahm3d) [inline css/listings.css:16]: Remove overflow.

File: css/listings.css [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+.listings__sidebar--header {
+  font-family: "Cinzel", serif;
+  font-size: 2.5rem;
+  font-weight: var(--font-bold);
+}
+
+.listings__sidebar--title {
+  font-family: "Cinzel", serif;
```
Comment (ttahm3d) [inline css/listings.css:30]: Move font family to class of the parent element.

**Review by ttahm3d — APPROVED** [review]
Have suggested changes. Please take a look.

**Pull request merged**